### PR TITLE
adding credential_download_generate_certificate

### PR DESCRIPTION
**Pull request by lwade**
3rd option so folks re-grabbing creds dont shoot themselves in the foot. ref: https://eucalyptus.atlassian.net/browse/EUCA-10520

### CODE REVIEW

File: faststart/ciab-template.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
       "ntp-server": "NTP",
       "system-properties": {
           "authentication.access_keys_limit": 10,
-          "authentication.signing_certificates_limit": 10
+          "authentication.signing_certificates_limit": 10,
+          "authentication.credential_download_generate_certificate": Limited
```
Comment (viglesiasce) [inline faststart/ciab-template.json]: This needs to be a quoted string.

**Comment by euca-qa-gui-tester** [comment]
Can one of the admins verify this patch?

**Comment by euca-qa-gui-tester** [comment]
Can one of the admins verify this patch?

**Comment by tbeckham** [comment]
Thanks @lwade

**Pull request merged**